superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -297,29 +313,38 @@ export default class AddSliceContainer extends React.PureComponent<
                   value={this.state.datasource}
                 />
                 <span>
-                  {t(
-                    'Instructions to add a dataset are available in the Superset tutorial.',
-                  )}{' '}
+                  <a
+                    href="/tablemodelview/list/#create"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    {t('Add a dataset')}
+                  </a>
+                  {t(' or ')}
```

Comment:
Spaces shouldn't be part of the translated string 😉

Reply:
Oops fixed!